exercises/concept/weighing-machine/.docs/hints.md

```diff
@@ -5,25 +5,29 @@
 - [Properties][docs.microsoft.com-properties]
 - [Using Properties][docs.microsoft.com-using-properties]
 
-## 1. Allow the weight to be set on the weighing machine
+## 1. Allow the weighing machine to have a precision.
 
-A property with a private [backing field][docs.microsoft.com-properties-with-backing-fields] is appropriate here.
+A get only property that can be intialised only from the constructor.
```

Comment:
Hints are usually listed as a list (this comment also applies to the other hints):

```suggestion
- A get only property that can be intialised only from the constructor.
```

While we're at it, maybe rephrase a little bit to be a bit more "directive":

```suggestion
- You need to add a `get`-only property to store the precision in.
- You can initialize the `get`-only property from the constructor (and only from the constructor).
```

Reply:
Is there perhaps a page to link to that describes a get-only property?

Reply:
@ErikSchierboom 
What do you think about this: https://stackoverflow.com/questions/2719699/when-should-use-readonly-and-get-only-properties